app/Http/Controllers/PostsController.php

```diff
@@ -97,6 +96,22 @@ public function index(Request $request)
             $query = $query->withoutAnonymousPosts();
         }
 
+        // This endpoint always returns posts in reverse chronological order. We'll
+        // therefore "force" the query string so that we can use it in `getCursor`.
+        // @TODO: There must be a more elegant way of doing this...
+        $query->orderBy('created_at', 'desc')
+            ->orderBy('id', 'asc');
```

Comment:
Why are there two `orderBy`s? In case there are identical timestamps on `created_at`?

Reply:
Yep!